bedrock/base/templates/base-error.html

```diff
@@ -0,0 +1,19 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends 'base-protocol.html' %}
+
+{% block page_title_prefix %}{% endblock %}
+{% block page_title_suffix %}{% endblock %}
+
+{% block body_class %}{% endblock %}
+
+{% block site_header %}
+  <header class="mzp-l-content header-spacing">
+    <div class="header-image">
+    {{ high_res_img('img/co-brand/firefox_moz_co-brand_sm.png', {'alt': 'Firefox | Mozilla'}) }}
```

Comment:
Might this sound better to a screen reader?

```suggestion
    {{ high_res_img('img/co-brand/firefox_moz_co-brand_sm.png', {'alt': 'Firefox, Mozilla'}) }}
```